### PR TITLE
Update README.md to address ZFS

### PR DESCRIPTION
**Pull request by kz6fittycent**
Addresses #2203

### CODE REVIEW

**Comment by kz6fittycent** [comment]
Retracting - this will NOT work!

**Comment by kz6fittycent** [comment]
I've included the steps I took to successfully point the snap to my zpool. All is well. Thanks @kyrofa  for your assistance with #2203

**Comment by pachulo** [comment]
Hi @kz6fittycent ! Thanks for your contribution.
The only doubt I have is if this belongs to the README or the Wiki: what do you think @kyrofa ?

**Comment by kyrofa** [comment]
Yeah I'm thinking wiki as well, but it will need to be one of us @pachulo.

**Comment by kz6fittycent** [comment]
> Hi @kz6fittycent ! Thanks for your contribution. The only doubt I have is if this belongs to the README or the Wiki: what do you think @kyrofa ?

Here's my argument FOR the README, instead of the wiki....how often to people read the README, let alone the wiki? :P 

Maybe a link to a special section on ZFS within the README to the wiki entry? I dunno - I would think either way, it'd be value-added.

**Comment by kyrofa** [comment]
Heh, how often will people read the README if it's as long as everything on the wiki? I think it's in our best interest to keep the README relatively brief. Not everyone uses ZFS.

**Comment by kz6fittycent** [comment]
> Heh, how often will people read the README if it's as long as everything on the wiki? I think it's in our best interest to keep the README relatively brief. Not everyone uses ZFS. So while this information is definitely useful to have (thank you!) it would just be noise to most people.

All good. Thanks for the inclusion. I'll leave it to ya'll.

File: README.md [hunk 1]
```diff
@@ -196,19 +196,31 @@ There are a few CLI utilities included:
     - Nextcloud data
 
 #### What about ZFS?
+
 - Zpools may be bind mounted to accomplish data access by the snap 
     - For example, create a new directory (e.g. `mkdir -p /mnt/zpool_nc`)
     - Depending on where your zpool exists (e.g. `/pool/tank01`), you can create a
       bind mount to allow the snap to acces/manage your data
-        - For example: `mount --bind /mnt/zpool_nc /pool/tank01`
+        - For example: `mount bind -o /mnt/zpool_nc /pool/tank01`
```
Comment (y0va) [inline README.md:204]: This is incorrect. It should be 

`mount --bind /pool/tank0 /var/snap/nextcloud/common/nextcloud/data`

also see: https://github.com/nextcloud-snap/nextcloud-snap/issues/2322

**Comment by scubamuc** [comment]
not sure where to put this in the wiki, I've added it to FAQ's although its not frequently asked 🙄
https://github.com/nextcloud-snap/nextcloud-snap/wiki/FAQ's#q-how-can-i-connect-a-zfs-device

maybe https://github.com/nextcloud-snap/nextcloud-snap/wiki/Wiki-Change-data-directory would have been wiser... but at least its documented now.... finally

**Pull request closed without merging**